app/presenters/teams/filtered_list.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Teams
+  class FilteredList < ::FilteredList
+
+    def filter
+      if query.present?
+        return entries.where('name like ?', "%#{query}%")
+      end
+
+      entries
+    end
+
+    def query
+      @params[:q]
+    end
+
+    def ids
+      @params[:team_ids]
```

Comment:
Shouldn't it be _ids_? In the tests, this param _ids_.

Reply:
Updated as team_ids everywhere